charts/jx-build-controller/templates/deployment.yaml

```diff
@@ -61,6 +61,15 @@ spec:
 {{- range $pkey, $pval := .Values.env }}
           - name: {{ $pkey }}
             value: {{ $pval }}
+{{- end }}
+{{- if .Values.envSecrets }}
+{{- range $pkey, $pval := .Values.envSecrets }}
```

Comment:
Couldn't this be simplified with `envFrom`?

Reply:
Was thinking the same